guides/common/modules/ref_ipv6-and-ipv4-requirements.adoc

```diff
@@ -0,0 +1,43 @@
+[id="ipv6-and-ipv4-requirements"]
+= IPv6 and IPv4 requirements
+
+You can install {Project} in an IPv4 network or in an IPv6 network.
+ifdef::satellite[]
+Dual-stack {Project} installation that uses both IPv4 and IPv6 is not supported.
+endif::[]
+
+The following requirements apply to installations in an IPv4 network:
+
+* Ensure an IPv6 loopback is configured on the base system.
+The loopback is typically configured by default.
+Do not disable it.
+* Do not disable IPv6 in kernel by adding the `ipv6.disable=1` kernel parameter.
+
+The following requirements apply to installations in an IPv6 network:
+
+ifdef::satellite[]
+* Deploy an external HTTP proxy server that supports both IPv4 and IPv6.
+This is required because Red Hat Content Delivery Network distributes content only over IPv4 networks, therefore you must use this HTTP proxy to pull content into the {Project} on your IPv6 network.
+You must configure {Project} to use this dual-stack (supporting both IPv4 and IPv6) HTTP proxy as the default HTTP proxy.
```

Comment:
Just a heads up that this requirement will disappear in the next few months. The requirement came from the RH CDN being IPv4-only, but they will make that a dual stack deployment so IPv6-only deployments don't need it. https://issues.redhat.com/browse/SAT-27425 if you want to follow this.

After that is available, we will have the same note as for upstream: users may still pull in custom content that is IPv4-only.

Reply:
Do you know which version will have the fix? The Jira doesn't have that information.

Reply:
I'm not sure that is public information so see the private message.